packages/flutter_tools/test/application_package_test.dart

```diff
@@ -144,6 +144,12 @@ void main() {
       expect(data.packageName, 'io.flutter.examples.hello_world');
       expect(data.launchableActivityName, 'io.flutter.examples.hello_world.MainActivity2');
     }, overrides: noColorTerminalOverride);
+    testUsingContext('Parses manifest with a dist namespace', () {
```

Comment:
Even though this file is not consistent about it, it would be nice if you kept spaces between `testUsingContext` invocations

Reply:
Yes I agree - I've added new lines throughout this file now.